README.md

```diff
@@ -147,7 +153,7 @@ Search UI currently provides two Connectors:
 1. **Elastic App Search**: [search-ui-app-search-connector](packages/search-ui-app-search-connector)
 2. **Elastic Site Search**: [search-ui-site-search-connector](packages/search-ui-site-search-connector)
 
-The [example search experience](#search-ui) above uses the Elastic App Search Connector:
+The [example search experience](#search-ui) above mapContextToProps the Elastic App Search Connector:
```

Comment:
I think this shouldn't have been replaced :)

Reply:
🤦‍♂